airbyte-webapp/src/components/connectorBuilder/Builder/KeyValueListField.module.scss

```diff
@@ -0,0 +1,39 @@
+@use "scss/variables";
+@use "scss/colors";
+
+$removeButtonWidth: 20px;
+
+.inputContainer {
+  width: 100%;
+  display: flex;
+  gap: variables.$spacing-xl;
+  align-items: center;
+}
+
+.labeledInput {
+  flex: 1;
+  max-width: 400px;
+  display: flex;
+  gap: variables.$spacing-sm;
+  align-items: center;
+}
+
+.kvLabel {
+  color: colors.$grey;
```

Comment:
Could we use a darker grey here? This will be hard to read for the vision-impaired and bad-screen-havers:
<img width="362" alt="Screenshot 2022-12-15 at 15 40 18" src="https://user-images.githubusercontent.com/1508364/207889630-8beaa810-8fad-4be5-9b06-654b342e8083.png">